src/utils/constants.ts

```diff
@@ -120,20 +120,20 @@ const socialLogins: readonly SocialAuthOption[] = [
     hasRedirectUri: false,
     documentation: 'https://developers.google.com/identity/sign-in/web/sign-in',
   },
-  {
-    name: 'Discord',
-    verifier: 'discord',
-    hasClientSecret: false,
-    hasRedirectUri: false,
-    documentation: 'https://canary.discord.com/developers/applications',
-  },
   {
     name: 'Twitch',
     verifier: 'twitch',
     hasClientSecret: false,
     hasRedirectUri: false,
     documentation: 'https://dev.twitch.tv/docs/authentication#registration',
   },
+  {
+    name: 'Discord',
+    verifier: 'discord',
+    hasClientSecret: true,
+    hasRedirectUri: false,
+    documentation: 'https://canary.discord.com/developers/applications',
```

Comment:
I don't think the canary subdomain is required. Please check once.

https://canary.discord.com/developers/applications

```suggestion
    documentation: 'https://discord.com/developers/applications',
```

Reply:
Yeah it's not required, just checked